repl/src/main/scala/com/cloudera/livy/repl/SparkInterpreter.scala

```diff
@@ -83,6 +86,33 @@ class SparkInterpreter(conf: SparkConf) extends Interpreter {
       setContextClassLoaderMethod.setAccessible(true)
       setContextClassLoaderMethod.invoke(sparkIMain)
 
+      // With usejavacp=true, the Scala interpreter looks for jars under System Classpath. But it
+      // doesn't look for jars added to MutableURLClassLoader. Thus extra jars are not visible to
+      // the interpreter. SparkContext can use them via JVM ClassLoaders but users cannot import
+      // them using Scala import statement.
+      //
+      // For instance: If we import a package using SparkConf:
+      // "spark.jars.packages": "com.databricks:spark-csv_2.10:1.4.0"
+      // then "import com.databricks.spark.csv._" in the interpreter, it will throw an error.
+      //
+      // Adding them to the interpreter manually to fix this issue.
+      var classLoader = Thread.currentThread().getContextClassLoader
+      while (classLoader != null) {
+        if (classLoader.getClass.getSimpleName == "MutableURLClassLoader") {
```

Comment:
Could you match the whole class name, just in case?

Also, I know that's currently not used by interactive sessions, but Livy allows adding jars to running sessions (that's exposed in the REST API); it would be good at some point to allow that to work for interactive sessions too. No need to do anything here, but filing a bug would be nice.